test/test_examples_2d.jl

```diff
@@ -322,13 +322,18 @@ end
             n_steps_max = 1,
             initial_conditions = "initial_conditions_constant")
   end
-  @testset "parameters_rotor.toml one step" begin
+  @testset "parameters_rotor.toml" begin
     test_trixi_run(joinpath(EXAMPLES_DIR, "parameters_rotor.toml"),
-            l2   = [0.019195558428459367, 0.0444345270981607, 0.04329215391420532, 0.0, 0.07639332626939939, 0.0044777850800556485, 0.007266820549785791, 0.0, 0.0008422441423441393],
-            linf = [0.5171127312547106, 1.9513340343194656, 1.9404704272260034, 0.0, 2.993145341292035, 0.14424331829406456, 0.20606277176198587, 0.0, 0.06685635962044646],
-            n_steps_max = 1)
+            l2   = [1.251062755110083, 1.8146501210703296, 1.6947702119211985, 0.0, 2.286126233832582, 0.2142656906510094, 0.23453619782492716, 0.0, 0.003028222728287231],
+            linf = [10.472575582440642, 14.089807317509075, 15.561008992418031, 0.0, 16.731740428408308, 1.3219222046920664, 1.4167075971532137, 0.0, 0.08046487935486654],
+            t_end = 0.05, volume_flux = "flux_derigs_etal")
```

Comment:
If we only test this flux, shouldn't we simply update the setting in the parameters file?

Reply:
Yeah, we can do that. This was just an experiment to see if the volume flux would influence the Travis. Before I made the shift permanent I wanted everyone to see it